modules/ROOT/pages/gatekeeper.adoc

```diff
@@ -47,34 +41,38 @@ anypoint.platform.gatekeeper=flexible
 
 === Security Level Definitions
 
+You can apply one of the listed security levels to Gatekeepr:
+
 ==== False
 
-When this value is set, no security is applied. Unwelcome traffic may reach the APIs during the policies application process. This value is available only for backward compatibility in Mule 3.x. Use ‘disabled’ instead. `disabled` is not available in all 3.x releases. `false` is not available in Mule 4.x.
+When you set this value, Gatekeeper does not apply any security. With this setting, the API might receive unwanted traffic during the period while the policies are being applied. In Mule 3.x, this value is available only for backward compatibility. Therefore, use ‘disabled’ instead for all other scenarios. Note that the `disabled` value is not available in all Mule 3.x releases and the `false` value is not available in Mule 4.x releases.
 
 ==== Disabled
 
-When this value is set, no security is applied. Unwelcome traffic may reach the APIs during the policies application process.
+When you set this value, Gatekeeper does not apply any security. With this setting, the API might receive unwanted traffic during the period while the policies are being applied.
```

Comment:
This should use exactly the same first sentence as the previous value's definition (minus the versioning info).

Reply:
...but now I think we should pair the two rather than separating them....
See below.

Reply:
Done! Thanks!